We have to move logs from /var/log to /var/log/kea to allow running
under unprivileged user.

--- a/src/bin/keactrl/kea-ctrl-agent.conf.pre
+++ b/src/bin/keactrl/kea-ctrl-agent.conf.pre
@@ -67,7 +67,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-ctrl-agent.log"
+                "output": "@localstatedir@/log/kea/kea-ctrl-agent.log"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
--- a/src/bin/keactrl/kea-dhcp-ddns.conf.pre
+++ b/src/bin/keactrl/kea-dhcp-ddns.conf.pre
@@ -44,7 +44,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-ddns.log"
+                "output": "@localstatedir@/log/kea/kea-ddns.log"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
--- a/src/bin/keactrl/kea-dhcp4.conf.pre
+++ b/src/bin/keactrl/kea-dhcp4.conf.pre
@@ -428,7 +428,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-dhcp4.log"
+                "output": "@localstatedir@/log/kea/kea-dhcp4.log"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
--- a/src/bin/keactrl/kea-dhcp6.conf.pre
+++ b/src/bin/keactrl/kea-dhcp6.conf.pre
@@ -353,7 +353,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-dhcp6.log"
+                "output": "@localstatedir@/log/kea/kea-dhcp6.log"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
--- a/src/bin/keactrl/kea-netconf.conf.pre
+++ b/src/bin/keactrl/kea-netconf.conf.pre
@@ -69,7 +69,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-netconf.log"
+                "output": "@localstatedir@/log/kea/kea-netconf.log"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
--- a/src/bin/keactrl/keactrl.in
+++ b/src/bin/keactrl/keactrl.in
@@ -333,7 +333,7 @@
 # to the default file.
 if [ -z "${KEA_LOGGER_DESTINATION}" ]; then
     prefix=@prefix@
-    export KEA_LOGGER_DESTINATION=@localstatedir@/log/kea.log
+    export KEA_LOGGER_DESTINATION=@localstatedir@/log/kea/kea.log
 fi
 
 command=${1}
